### DISCUSSION TITLE
I'd like to join `rust-mobile`

### DISCUSSION THREAD

**Discussion by madsmtm** [rust-mobile/.github#6] [General]
Hey, I'm looking into the current Rust offers/crates for bundling and running applications on iOS, and... It's a mess. To this end, I'd like to join the `rust-mobile` organization, to begin rectifying some of this.

I'd start by contacting BrainiumLLC, the owners of `cargo-mobile`, and the maintainers of `cargo-dinghy`, and asking them to move to `rust-mobile`, since these, along with `xbuild` and `cargo-apk`, have a lot of overlap, so I think it makes sense to maintain them together in the same organization.

We'd also want some collaboration with the [`dodorare`](https://github.com/dodorare) organization.

CC @rib, @rust-mobile

**Comment by rib** [comment]
Hey, @madsmtm, thanks for reaching out -  these all sound like relevant initiatives and considering knowing that you've already done lots of good work in the area of iOS enabling for Rust I'm more than happy to add you as a member here. 👍

**Reply by madsmtm** [reply]
Wonderful! I'll keep this discussion open to post status updates, to begin with I've opened https://github.com/BrainiumLLC/cargo-mobile/issues/128.

**Reply by madsmtm** [reply]
I finally got an answer from Rory Snively via. Francesca Lovebloom via. email; Brainium does not want to give up ownership of `cargo-mobile` (which is unfortunate, but totally fair).

I'll try to move forwards with some of the other projects.

**Reply by MarijnS95** [reply]
Did you see that tauri folks forked it into `cargo-mobile2`?

**Comment by npetrangelo** [comment]
Any updates on how the iOS user story has evolved since this post?

**Reply by madsmtm** [reply]
Not really, no, I got side-tracked a bunch with [making `rustc` work nicer with Xcode](https://github.com/rust-lang/rust/issues/129432), and [making it easier to use Mac Catalyst](https://blog.rust-lang.org/2024/10/17/Rust-1.82.0.html#mac-catalyst-targets-are-now-tier-2).
